docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Filecoin FAQ
+description: Filecoin frequently asked questions.
+---
+
+# Filecoin FAQ
+
+[[toc]]
+
+## Getting Started
+
+### How do ordinary people, not geeks, get involved in this work?
+
+You can start from checking out the tutorials from [Protoschool](https://proto.school/#/tutorials)
+Participate in the hackathon and future events of [Filecoin Ignite](https://ignite.fil.events/)
+Check out the different tools to [build dApps on Filecoin](https://blog.textile.io/)
+
+### What are some of the primary use cases for Filecoin at mainnet launch?
+
+The primary use case for Filecoin at first will be more 'long term' storage applications. Archival, personal backups, etc. As well as applications like existing dApps that don't want to (or 'can't') store their data in traditional ways. There is fairly significant demand among current decentralized applications for a solution to how their data should be stored, and a good number of them already use IPFS (but resort to pinning data themselves, on their servers to ensure the data sticks around). These apps already have wallets built in usually, and adding some sort of micropayment for the data they use isn't too big of a stretch.
```

Comment:
Replace with:

Filecoin is a protocol that provides core primitives enabling a truly trustless decentralized storage network. These primitives and features include cryptographic storage proofs (that are publicly verifiable so you can trust that when a miner says they're storing your data, they actually are), cryptoeconomic mechanisms (see our recent description of Filecoin's cryptoeconomics [here](https://filecoin.io/blog/filecoin-cryptoeconomic-constructions/)), and a public blockchain. Filecoin provides these primitives to solve the really hard problem of creating a truly trustless decentralized storage network.

On top of the core Filecoin protocol, there are a number of "layer 2 solutions" -- many incorporating [IPFS](https://ipfs.io) -- that enable a broad array of use cases and applications. These solutions include Powergate, Textile Hub, and more. Using these solutions, any use case that can be built on top of IPFS can also be built on Filecoin.

Some of the primary areas for development expected on Filecoin around mainnet launch are: 
- Additional developer tools and layer 2 solutions/libraries that strengthen Filecoin as a developer platform and ecosystem
- IPFS apps that rely on decentralized storage solutions and want a decentralized data persistence solution as well
- Financial tools and services on Filecoin, e.g. wallets, signing libraries, and more
- Beginnings of verified storage solutions -- applications that use Filecoin's publicly verifiable cryptographic proofs in order to provide trustless and timestamped guarantees of storage to their users